kolibri/auth/management/commands/syncdata.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from clint.textui.progress import Bar as ProgressBar
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from requests_toolbelt import MultipartEncoder, MultipartEncoderMonitor
+
+DB_PATH = settings.DATABASES["default"]["NAME"]
+
+CENTRAL_SERVER_DB_UPLOAD_URL = "http://kolibridataupload.learningequality.org/upload/"
+
+def create_callback(encoder):
+    encoder_len = encoder.len
+    bar = ProgressBar(expected_size=encoder_len, filled_char="=")
+
+    def callback(monitor):
+        bar.show(monitor.bytes_read)
+
+    return callback
+
+class Command(BaseCommand):
+    help = "Uploads the local database to a central server for backup and reporting"
+
+    def handle(self, *args, **options):
+
+        self.stdout.write("Uploading database to central server...\n")
+
+        encoder = MultipartEncoder({"file": ("db.sqlite3", open(DB_PATH, "rb"), "application/octet-stream")})
+        monitor = MultipartEncoderMonitor(encoder, create_callback(encoder))
+        r = requests.post(CENTRAL_SERVER_DB_UPLOAD_URL, data=monitor, headers={"Content-Type": monitor.content_type})
```

Comment:
As mentioned in person after, it will error out if the DB isn't the right format, and stores it based on the dataset ID within the DB.